src/data/markdown/docs/30 xk6-browser/01 Get started/02 Running xk6-browser.md

```diff
@@ -118,11 +120,11 @@ To avoid timing errors or other race conditions in your script, if you have acti
 
     // Enter login credentials and login
     page.locator('input[name="login"]').type('admin');
-    page.$('input[name="password"]').type('123');
+    page.locator('input[name="password"]').type('123');
 
     // Wait for asynchronous operations
     // eslint-disable-next-line no-undef
-    Promise.all([page.waitForNavigation(), page.$('input[type="submit"]').click()])
+    Promise.all([page.waitForNavigation(), page.locator('input[type="submit"]').click()])
```

Comment:
The page locator does not support async click yet :-( So it is valid to use `page.$` here with a comment explaining the locator.click does not yet support async click.

Reply:
Oh, I didn't know that, thanks for clarifying! Will change it back 👍